lib/doorkeeper/models/application_mixin.rb

```diff
@@ -12,19 +10,7 @@ module ApplicationMixin
     include Models::Scopes
 
     included do
-      # Use BCrypt as the hashing function for applications
-      self.secret_hash_function = lambda do |plain_token|
-        BCrypt::Password.create(plain_token.to_s)
-      end
-
-      # Also need to override the comparer function for BCrypt
-      self.secret_comparer = lambda do |plain, secret|
-        begin
-          BCrypt::Password.new(secret.to_s) == plain.to_s
-        rescue BCrypt::Errors::InvalidHash
-          false
-        end
-      end
+      set_bcrypt_hashing_if_available
```

Comment:
I prefer to use something more abstract, like `configure_secrets_hashing`, because we can replace bcrypt or add other engines for hashing, and the name of the method ceases to comply with the purposes.

Reply:
Yes, good point!